Bottom: 511d59a4118b97b791c3953b1998887da29858e2
Top:    bcdcf7f7bde15e2c6e6ea0ddbe0cf8fc11348fdf
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-16 15:13:56 +0800

Refresh of autofs-5.0.7-only-prove-specific-nfs-version-if-requested.patch

---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index a1e2d40..82f674d 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -149,7 +149,7 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 				else if (strncmp("vers=3", cp, o_len) == 0 ||
 					 strncmp("nfsvers=3", cp, o_len) == 0) {
 					vers &= ~NFS4_VERS_MASK;
-					vers = (NFS3_REQUESTED & ~NFS_VERS_MASK);
+					vers |= (NFS3_REQUESTED & ~NFS_VERS_MASK);
 				} else if (strncmp("vers=2", cp, o_len) == 0 ||
 					 strncmp("nfsvers=2", cp, o_len) == 0) {
 					vers &= ~NFS4_VERS_MASK;
